### ISSUE TITLE
Open links to the normal web into a seperate browser?

### ISSUE THREAD

**Issue by Ravenbird** [beakerbrowser/beaker#841]
Is there a way to open links to external websites in another browser? For normal websites I prefer a browser with appropriate extensions.

**Comment by pfrazee** [comment]
Not currently. Like, you'd want https links to open in another browser?

**Comment by Ravenbird** [comment]
Normal web is full of crab for what you need a adblocker and other extensions on firefox. I can also copy the urls, but thats not so handy.

**Comment by pfrazee** [comment]
Yeah I feel that. We *do* have a minimal ad blocker built into the 0.8 release, fwiw! But yeah, I can imagine the value of this feature until our regular browsing is solid. We'll slate this as a feature request.

**Comment by Ravenbird** [comment]
Thank you very much. Keep on good work! :-)

**Comment by pfrazee** [comment]
Thanks!
